assemblies/assembly_filtering-process-instances/main.adoc

```diff
@@ -0,0 +1,24 @@
+[id='assembly_filtering-process-instances']
+
+include::_artifacts/document-attributes.adoc[]
+
+= Filtering process instances
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:doctype: book
+:imagesdir: _images
+:context: filtering-process-instances
+include::_artifacts/author-group.adoc[]
+
+// Purpose statement for the assembly
+In {PRODUCT}, you can view all process instances from the *Manage Process Instances* page. As an admin, you can filter through process instances in {CENTRAL}.
+
+.Prerequisites
```

Comment:
Hi Gaurav, the style for prerequisites is more of a passive wording, such as:

Prerequisites

Red Hat JBoss Enterprise Application Platform 7.1.0 is installed. See Red Hat JBoss EAP 7.1.0 Installation Guide.
Red Hat Process Automation Manager is installed. For more information, see Installing and configuring Red Hat Process Automation Manager on Red Hat JBoss EAP 7.1.
Red Hat Process Automation Manager is running and you can log in to Business Central with the admin role.

Not sure why, but that's what I was told :)

Reply:
Changed as suggested.